Readme.md

```diff
@@ -0,0 +1,494 @@
+[1]: https://www.mongodb.com/atlas/database
+
+[2]: https://www.mongodb.com/docs/atlas/sample-data/
+
+[3]: https://www.mongodb.com/cloud/atlas/register
+
+[4]: https://www.mongodb.com/docs/drivers/kotlin/coroutine/current/
+
+[5]: https://github.com/mongodb-developer/kotlin-driver-quick-start
+
+# Getting started with MongoDB for building backend Applications using the official Kotlin driver
+
+> This is an introduction article on how to build backend application in Kotlin using official MongoDB Kotlin driver and [MongoDB Atlas][1].
```

Comment:
Rather than adding this line, can you give the article a proper introduction. Some ideas:

- Identify a problem that only Kotlin can solve and explain how this tutorial will help the reader solve that problem.
- Explain why you love Kotlin so much and highlight some of the things you're going to accomplish in the article.

Yes some readers will go straight to the code they can copy and paste, but others need a hook to know why they are going to continue reading. I personally fall into category 2.